### DISCUSSION TITLE
How do I solve this deployment error

### DISCUSSION THREAD

**Discussion by oyedejioyewole** [vercel/vercel#5194] [Help]
Hello, how do I solve this error I get after successful uploading of a project from the console online
I use django and I think I have configured everything properly but the page doesn't render out instead I get this error

![Error Picture](https://user-images.githubusercontent.com/57059138/93118432-3c0b2480-f6b8-11ea-8b22-c309d9f3dad8.png)

`The logs are below but like 5 times worse`

![Screenshot_37](https://user-images.githubusercontent.com/57059138/93118824-c489c500-f6b8-11ea-9f4b-0d67e144ef6d.png)

**General message in logs are**
`No module named 'website_viewer': ModuleNotFoundError`

But in the directory structure which the `website_viewer` folder is found in `src/website_viewer`, which is exactly what I configured in the `vercel.json` file

```json
{
  "version": 2,
  "name": "website-peeker",
  "builds": [
    {
      "src": "src/website_viewer/wsgi.py",
      "use": "@ardnt/now-python-wsgi",
      "config": {
        "maxLambdaSize": "30mb"
      }
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "src/website_viewer/wsgi.py"
    }
  ]
}
```

The `website_viewer` folder is the main project folder

**Comment by paulogdm** [comment]
You may need to contact the author of `@ardnt/now-python-wsgi` since this doesn't appear to be a platform-specific issue.

Alternatively, you can use `@vercel/python`.
